packages/manager/apps/container/src/payment-modal/PaymentModal.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState, useEffect } from 'react';
+
+import { Modal } from 'react-bootstrap';
+import { useTranslation } from 'react-i18next';
+
+import { useShell } from '@/context';
+
+import { fetchIcebergV6 } from '@ovh-ux/manager-core-api';
+
+import './styles.scss';
+
+interface IPaymentMethod {
+  icon?: any;
+  label: string;
+  status: string;
+  default: boolean;
+  oneclick: boolean;
+  lastUpdate: string;
+  description: string;
+  integration: string;
+  paymentType: string;
+  creationDate: string;
+  paymentMeanId: number;
+  expirationDate?: string;
+  paymentSubType: string;
+  paymentMethodId: number;
+}
+
+interface IPaymentMethodResponse {
+  data: IPaymentMethod[];
+}
+
+const PaymentModal = (): JSX.Element => {
+  const NO_DEFAULT_CARD = 'NO_DEFAULT_CARD';
+  const EXPIRED_CARD = 'EXPIRED_CARD';
```

Comment:
This could be extracted into a constant / enum (not sur how these are managed in the react stack)

Reply:
Since it's just internval vars, it shouldn't need to be a in separate constants file, but I extracted it and did like it's done in other react components.